cmds/bitstream.c

```diff
@@ -31,7 +31,7 @@ static int cmd_bitstream(int argc, char *argv[])
 	ssize_t res;
 	handler_t handler;
 	u8 buff[SIZE_MSG_BUFF];
-	addr_t offs = 0, addr = BITSREAM_ADDR;
+	addr_t offs = 0, addr = ADDR_BITSREAM;
```

Comment:
`BITSTREAM` perhaps?

Reply:
Fixed. I found another typo: `MICORN_MT25QL01GBBB` changed to `MICRON_MT25QL01GBBB`

Reply:
All appearances had this typo